src/utils/api.js

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+const api = axios.create({ baseURL: "https://nc-news-qvv1.onrender.com/api" })
+
+export const getArticles = () => {
+    return api.get('/articles')
+    .then(({ data: {articles}}) => {
+        return articles
+    })
+} 
```

Comment:
Great that you've separated out this functionality into a separate API js file and nice use of axios ⭐